src/main/java/uk/gov/hmcts/reform/refunds/services/IdamServiceImpl.java

```diff
@@ -33,6 +36,9 @@
     public static final String USER_FULL_NAME_ENDPOINT = "/api/v1/users";
     private static final Logger LOG = LoggerFactory.getLogger(IdamServiceImpl.class);
     static private final String LIBERATA_NAME = "Middle office provider";
+    private static final String INTERNAL_SERVER_ERROR = "Internal Server error. Please, try again later";
+    private static final int USER_INFO_SIZE = 100;
```

Comment:
Move this to yaml file and change the value to 300

Reply:
Done